packages/dds/tree/src/test/simple-tree/api/schemaFactory.spec.ts

```diff
@@ -738,6 +747,15 @@ describe("schemaFactory", () => {
 			const factory = new SchemaFactoryAlpha("test");
 			class NamedRecord extends factory.record("name", factory.number) {}
 			const namedInstance = new NamedRecord({ x: 5 });
+			const x: number = namedInstance.x;
+			// TODO: this should not compile as the typing is incorrect (y is undefined, not number)
```

Comment:
Wouldn't this apply to the line above as well? `namedInstance` is typed as `NamedRecord`, I don't think the compiler can tell that `x` is present. Right?